Model/TrackPluginInstrumentation.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Razorpay\Magento\Model;
+
+use Razorpay\Api\Api;
+use Magento\Framework\Module\ModuleListInterface;
+use Razorpay\Api\Errors;
+use Razorpay\Magento\Model\PaymentMethod;
+use Razorpay\Magento\Model\Config;
+
+use function PHPSTORM_META\type;
+
+class TrackPluginInstrumentation
+{
+    const MODULE_NAME = 'Razorpay_Magento';
+
+    protected $api;
+
+    protected $mode;
+
+    protected ModuleListInterface $moduleList;
```

Comment:
@nikhilsharma1100 php error thrown at this line, please check its use and execution.

Reply:
removed typed declaration